src/extension.ts

```diff
@@ -87,6 +87,16 @@ export async function activate(context: vscode.ExtensionContext) {
         async (node?: RegionNode) => await ext.awsContextCommands.onCommandHideRegion(safeGet(node, x => x.regionCode))
     )
 
+    // register URLs in extension menu
+    vscode.commands.registerCommand(
+        'aws.help',
+        () => { vscode.env.openExternal(vscode.Uri.parse('https://aws.amazon.com/visualstudiocode/'))}
```

Comment:
Do we have a file of constants to add the URLs to?

Reply:
Yep, see `src/lambda/constants.ts` and `src/shared/constants.ts`.

Reply:
Can you verify that these URLs will redirect to the user's locale if not us-EN? The docs aren't translated yet, so just make sure it redirects to (for example) `https://aws.amazon.com/es/visualstudiocode/` for Spanish.

Reply:
I'll definitely add this to the constants file, but otherwise I'll see if I can add some sort of language support.

Reply:
Great--I'd be surprised if it doesn't redirect already.

Reply:
@mpiroc I am tentative about adding full languages for now. We want this to link to documentation (instead of the product page), and that probably won't be translated in one fell swoop. This means that we'd link to (untranslated) product pages, and then rip it all out when we get the English documentation.

I have a switch statement in place that we can add to as we receive documentation translations...I'm just not 100% sure if we should add languages before then.

Reply:
@bryceitoc9 Sorry for the confusion--this doesn't require any code changes, I just meant to verify the redirect in your browser. I set my browser settings to Spanish, and visited `https://aws.amazon.com/visualstudiocode/`. As expected, it redirected to `https://aws.amazon.com/es/visualstudiocode/`. So we're good, no action required.